mobile/lib/screens/web_view_page.dart

```diff
@@ -1,11 +1,19 @@
 import 'dart:async';
+import 'dart:io';
 
 import 'package:app/themes/theme.dart';
 import 'package:app/utils/utils.dart';
 import 'package:app/widgets/widgets.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:webview_flutter/webview_flutter.dart';
 
+class LoadingCubit extends Cubit<int> {
+  LoadingCubit() : super(0);
+
+  void setProgress(int progress) => emit(progress);
+}
```

Comment:
Cubit to handle loading progress.
Reference:
https://pub.dev/packages/flutter_bloc